app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -724,64 +747,67 @@
 
     <!-- App Setting: Show Equinox / Solstice -->
-    <string name="configLabel_ui_showEquinox">Mostrar solstícios / equinócios</string>
-    <string name="configLabel_ui_showEquinox_summary">Mostrar tempo restante pro próximo solstício e/ou equinócio.</string>
-    <string name="configLabel_ui_showEquinox_notImplemented">Solstício e equinócio não implementados nesse banco de dados.</string>
+    <string name="configLabel_ui_showEquinox">Exibir Solstício / Equinócio</string>
+    <string name="configLabel_ui_showEquinox_summary">Exibe tempo até o próximo solstício ou equinócio.</string>
 
     <!-- App Setting: Solstice Tracking Mode -->
-    <string name="configLabel_ui_solsticeTrackingMode">Procurar Solstício</string>
+    <string name="configLabel_ui_solsticeTrackingMode">Localizar Solstício</string>
     <string name="configLabel_ui_solsticeTrackingMode_summary"><xliff:g id="modeName" example="Upcoming Event">%s</xliff:g></string>
 
     <!-- App Setting: show data source -->
-    <string name="configLabel_ui_showdatasource">Mostrar Banco de Dados</string>
-    <string name="configLabel_ui_showdatasource_summary">Mostrar uma área indicando a atual configuração.</string>
+    <string name="configLabel_ui_showdatasource">Exibir Fontes de Dados</string>
+    <string name="configLabel_ui_showdatasource_summary">Exibe um rótulo indicando a atual configuração.</string>
 
     <!-- App Setting: show golden hour -->
-    <string name="configLabel_ui_showgoldhour">Mostrar Hora Dourada</string>
-    <string name="configLabel_ui_showgoldhour_summary">Mostrar horários de hora dourada.</string>
+    <string name="configLabel_ui_showgoldhour">Hora Dourada</string>    <!-- TODO -->
+    <string name="configLabel_ui_showgoldhour_summary">Exibe momentos de hora dourada.</string>
 
     <!-- App Setting: show blue hour -->
-    <string name="configLabel_ui_showbluehour">Mostrar Hora Azul</string>
-    <string name="configLabel_ui_showbluehour_summary">Mostrar horários de hora azul.</string>
+    <string name="configLabel_ui_showbluehour">Hora Azul</string>  <!-- TODO -->
+    <string name="configLabel_ui_showbluehour_summary">Exibe momentos de Hora Azul.</string>
 
     <!-- App Setting: UI Theme -->
-    <string name="appThemes_lightTheme">Tema Claro</string>
+    <string name="appThemes_lightTheme">Tema Branco</string>
     <string name="appThemes_darkTheme">Tema Escuro</string>
     <string-array name="appThemes_display"> <!-- display values (one-to-one with appThemes_values) -->
         <item>@string/appThemes_lightTheme</item>
         <item>@string/appThemes_darkTheme</item>
-        <item>Claro / Escuro (Modo&#160;Noite)</item>  <!-- TODO: nbsp -->
+        <item>Tema Branco / Tema Escuro (Modo&#160;Noite)</item>  <!-- TODO: nbsp -->
+    </string-array>
+    <string-array name="appThemes_values" translatable="false">  <!-- config values (one-to-one with appThemes_display) -->
```

Comment:
These are actually enum values (anything marked translatable="false" should only exist once in the default strings).

It is possible to translate the labels though, as you've done a few lines above with `appThemes_lightTheme`.